jvm-agent/src/test/resources/analysis/controlflow/golden/loops/doWhileLoopContinueBreak.txt

```diff
@@ -0,0 +1,17 @@
+LOOPS
+LOOP 1
+  HEADER: B1
+  BODY: B1, B2, B3, B5, B6
+  BACK EDGES:
+    B6 -> B1 : JUMP(opcode=IF_ICMPLT)
+  NORMAL EXITS:
+    B3 -> B4
+    B6 -> B7
+  EXCEPTION EXIT HANDLERS: NONE
+
+LOOPS BY BLOCK
```

Comment:
Do we need to print this info?
Alternatively, we can just check right away in the tests that this bb-to-loop mapping is consistent with the loop-to-bbs mapping.